smoke/internal/testdata/project.yml

```diff
@@ -128,7 +128,7 @@ tasks:
       - command: archive.targz_extract
         params:
           path: "archive.tgz"
-          destination: "output"
+          destination: "output/targz"
```

Comment:
This is necessary due to the comment about conflicting files:

> the behavior of v2 was to overwrite/continue on error if it tried to extract a file that already exists with the same name, but now it stops if it encounters an existing file that conflicts

Since all the archive files contains the same files, all of these extracts were extracting the same files into the same directory. This tests that each individual archived file can be extracted into file paths that don't conflict.

Reply:
Ah I see now, thanks!